Maps.php

```diff
@@ -47,8 +47,7 @@
 $GLOBALS['wgExtensionMessagesFiles']['MapsMagic'] = __DIR__ . '/Maps.i18n.magic.php';
 $GLOBALS['wgExtensionMessagesFiles']['MapsAlias'] = __DIR__ . '/Maps.i18n.alias.php';
 
-
-$GLOBALS['wgExtensionFunctions'][] = function () {
+$GLOBALS['wgExtensionFunctions'][] = function() {
```

Comment:
In common MediaWiki style there is space after anonymous functions, just like with `if` to indicate it's not a function call.

Reply:
Thanks. Will change this. Seems like we have a slight skew to the less spacy form in the FUN code, but the usage is inconsistent, and we apparently don't have many anon functions.

Reply:
Actually this is weird in MW style then...

    function noSpaceHere() {

vs

    $whySpace = function () {

So in case of anon declarations you have a space and for named ones you do not.

Reply:
Just a different point of view: `function` is a keyword and there is a space after keyword.

Reply:
Right. Thanks for explaining